codec-http2/src/main/java/io/netty/handler/codec/http2/DefaultHttp2HeadersDecoder.java

```diff
@@ -53,14 +48,15 @@ public DefaultHttp2HeadersDecoder() {
     }
 
     public DefaultHttp2HeadersDecoder(boolean validateHeaders) {
-        this(DEFAULT_MAX_HEADER_SIZE, DEFAULT_HEADER_TABLE_SIZE, validateHeaders);
+        this(DEFAULT_MAX_HEADER_SIZE, DEFAULT_HEADER_TABLE_SIZE, validateHeaders, 32);
     }
 
-    public DefaultHttp2HeadersDecoder(int maxHeaderSize, int maxHeaderTableSize, boolean validateHeaders) {
+    public DefaultHttp2HeadersDecoder(int maxHeaderSize, int maxHeaderTableSize, boolean validateHeaders,
+                                      int initialCapacity) {
```

Comment:
consider adding comments for `initialCapacity` and/or improving the variable name (e.g `initialHuffmanDecodeCapacity`)